index.bs

```diff
@@ -2001,6 +2027,68 @@ The [=remote end steps=] with |session| and |command parameters| are:
 
 </div>
 
+#### The session.installWebExtension Command #### {#command-session-installWebExtension}
+
+The <dfn export for=commands>session.installWebExtension</dfn> command installs
+a web-extension for a given session.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+     <pre class="cddl remote-cddl">
+     session.InstallWebExtension = (
+       method: "session.installWebExtension",
+       params: session.InstallWebExtensionParameters
+     )
+
+     session.InstallWebExtensionParameters = {
+       extension: session.Extension
+     }
+     </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+      <pre class="cddl">
+      session.InstallWebExtensionResult = (
```

Comment:
You don't need to manually define success and failure types; you can just define an error code and either `return [=success=] with data [map matching success type]` or `return error with error code [new code]` and that will handle this.